CHANGELOG.mkd

```diff
@@ -1,12 +1,20 @@
 # OpenXC iOS App Demo Changelog
 
-## v5.0.0
+## v6.0.0
+
+* Improvement: Support for Protobuf 3
+* Feature: Added disable trace loop and restart trace features
+* Feature: Added base64 encoding to source ID for cloud export
```

Comment:
Change this to" Feature:Upload Trace to URL"

Reply:
I updated the change log